src/kedro_templar/process_templates.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import os
+from pathlib import Path
+from typing import Text, Dict
+import boto3
+from botocore.exceptions import ClientError
+import click
+import yaml
+from jinja2 import Environment, FileSystemLoader
+
+log = logging.getLogger(__name__)
+
+
+# settings.py
+TEMPLATES_DIR = Path('./examples/base')
```

Comment:
Move to separate file `src/kedro_templar/settings.py`

Reply:
ok